README.md

```diff
@@ -83,14 +83,29 @@ myCustomAction(firstParam, secondParam, label) {
 
 # API
 
-`new SetupChain([Object], [Object])`
+## `new SetupChain(state, actions)`
```

Comment:
yeah I like this better, the param names are clearer here with docs below on the type